.github/workflows/gem5.yml

```
@@ -142,3 +142,33 @@ jobs:
           cd $GEM5_HOME/util/xs_scripts/test_h
           bash ../kmh_6wide_h.sh /nfs/home/share/gem5_ci/checkpoints/gcbh_test.zstd
 
+  raw_linux_boot_test:
+    runs-on: [self-hosted, open]
+    continue-on-error: false
+    timeout-minutes: 10
+    name: XS-GEM5 - Boot raw Linux
+    steps:
+      - uses: actions/checkout@v2
```

actionlint (1.7.11)

[error] the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)